R/utilities.R

```diff
@@ -3049,21 +3046,18 @@ make_igv_snapshot = function(bams,
 #' @param text_size Size of the text on the forest plot of differentially enriched CNV. Default text-size is 7.
 #' @param blacklisted_regions Optionally, specify any descriptors (value from column `Descriptor` of GISTIC2.0 all_lesions output file) to filter out before any comparisons are done. It is possible to specify a list of multiple descriptors, for example, c("3p12.3", "12p13.2"). Default is NULL.
 #'
-#' @return list of 3 objects:
```

Comment:
Did this cause an error in Git Actions? if not, wouldn't it make more sense to actually describe all the elements of the list?

Reply:
Yes it was making the build fail. The returns should be an explicit object (data frame, list, integer, string etc)